sources/exegol/load_supported_setups.sh

```diff
@@ -124,7 +144,8 @@ deploy_tmux
 deploy_vim
 deploy_apt
 deploy_python3
+deploy_firefox_addons
```

Comment:
Dont forget to submit a PR to the exegol-docs repository to add and explain your new feature.
@ShutdownRepo to review how we manage this requirement

Reply:
Indeed, I didn't think about this. @Skileau happy to help if needed. Additions need to be made at [`source/exegol-image/my-resources.rst`](https://github.com/ThePorgs/Exegol-docs/blob/main/source/exegol-image/my-resources.rst).

Reply:
Yes, I had a quick look at how the doc is written this morning.
I will add a part for each of my recent PRs ;)

Reply:
➡️ https://github.com/ThePorgs/Exegol-docs/pull/6
Do not hesitate to request changes, I tried to keep the other section's style.

Reply:
Picked up at https://github.com/ThePorgs/Exegol-docs/pull/7